### PR TITLE
refactor: OCR (Batch/Online Processing) codelab updates

### PR DESCRIPTION
**Pull request by holtskinner**
- Created Separate Examples for processing a list of specific documents and a full directory
- Changed Document Retrieval from GCS to use information from `BatchProcessMetadata`

### CODE REVIEW

File: community/codelabs/docai-ocr/batch_processing.py [hunk 1]
```diff
@@ -1,69 +1,32 @@
 # type: ignore[1]
+# pylint: skip-file
```
Comment (galz10) [inline community/codelabs/docai-ocr/batch_processing.py:2]: does this cause pylint to skip this whole file ?

File: community/codelabs/docai-ocr/batch_processing.py [hunk 2]
```diff
@@ -1,69 +1,32 @@
...
-    return output_documents
-
-
 PROJECT_ID = "YOUR_PROJECT_ID"
 LOCATION = "YOUR_PROJECT_LOCATION"  # Format is 'us' or 'eu'
 PROCESSOR_ID = "YOUR_PROCESSOR_ID"  # Create processor in Cloud Console
 
 # Format 'gs://input_bucket/directory/file.pdf'
-GCS_INPUT_URI = "INPUT_BUCKET_URI"
+GCS_INPUT_URI = "gs://cloud-samples-data/documentai/codelabs/ocr/Winnie_the_Pooh.pdf"
```
Comment (galz10) [inline community/codelabs/docai-ocr/batch_processing.py:20]: will someone be able to access this gcs bucket ?

File: community/codelabs/docai-ocr/batch_processing_directory.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+from google.cloud import documentai_v1 as documentai
+from google.cloud import storage
+
+
+PROJECT_ID = "YOUR_PROJECT_ID"
+LOCATION = "YOUR_PROJECT_LOCATION"  # Format is 'us' or 'eu'
+PROCESSOR_ID = "YOUR_PROCESSOR_ID"  # Create processor in Cloud Console
+
+# Format 'gs://input_bucket/directory'
+GCS_INPUT_PREFIX = "gs://cloud-samples-data/documentai/codelabs/ocr/multi-document"
```
Comment (galz10) [inline community/codelabs/docai-ocr/batch_processing_directory.py:20]: same question is this publicly available

File: community/codelabs/docai-ocr/batch_processing.py (hunk 2 above)
Comment (holtskinner) [reply community/codelabs/docai-ocr/batch_processing.py:20]: Yes, this is fully publicly accessible (`allUsers` has read permission)

File: community/codelabs/docai-ocr/batch_processing.py (hunk 1 above)
Comment (holtskinner) [reply community/codelabs/docai-ocr/batch_processing.py:2]: Yes, Changing to just disable pylint on the line causing issues

File: community/codelabs/docai-ocr/batch_processing_directory.py (hunk 3 above)
Comment (holtskinner) [reply community/codelabs/docai-ocr/batch_processing_directory.py:20]: Yes, allUsers have read access

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**